src/IssueViz.Security.UnitTests/Helpers/ThreadHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FluentAssertions;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.Helpers
+{
+    internal class ThreadHelper
```

Comment:
The alternative was adding shell tricks to TestsInfrastructure and the other test projects.